### DISCUSSION TITLE
只输入本地ip 的话怎么样才能跳转到正确的重定向页面 报错 缺少 basename

### DISCUSSION THREAD

**Discussion by anhang-cmyk** [umijs/umi#11106] [Q&A]
![image](https://user-images.githubusercontent.com/75460955/236684569-d4937aa7-d951-4578-8b16-447a26964be9.png)
![image](https://user-images.githubusercontent.com/75460955/236684586-1c47cdfc-eb40-4b43-bda2-5fabcf8f73d1.png)

**Comment by fz6m** [comment]
需要给一个最小复现。
